src/nsresourced/nsresourcework.c

```diff
@@ -282,7 +282,7 @@ static int vl_method_get_group_record(sd_varlink *link, sd_json_variant *paramet
                 if (offset >= userns_info->size) /* Outside of range? */
                         goto not_found;
 
-                if (gid_is_valid(p.gid) && p.uid != userns_info->start + offset)
+                if (gid_is_valid(p.gid) && p.gid != userns_info->start_gid + offset)
```

Comment:
I think it is better to fix the typo `p.uid` -> `p.gid` in a separate commit, as it was introduced by 8aee931e7ae (v256), and the fix should be backported to stable.

Reply:
Will submit as a separate commit.

Reply:
-> #36723.